kolibri/plugins/learn/assets/src/views/thumbnails/ChannelThumbnail.vue

```diff
@@ -10,14 +9,12 @@
       class="image"
       :src="thumbnailUrl"
       alt=""
-    >
+      loading="lazy"
+    />
```

Comment:
This is something else that is explicitly disallowed by our linting.

HTML void tags should not be self closing, because self closing tags do not exist in HTML: https://developer.mozilla.org/en-US/docs/Glossary/Void_element